terra-app-setup-chart/values.yaml

```diff
@@ -19,4 +19,9 @@ relaylistener:
   removeEntityPathFromHttpUrl: "true"
   leoUrl: "https://leonardo.dsde-dev.broadinstitute.org"
   workspaceId: RUNTIME_PARAMETER
-  runtimeName: RUNTIME_PARAMETER
\ No newline at end of file
+  runtimeName: RUNTIME_PARAMETER
+  validHosts:
+    - RUNTIME_PARAMETER
+    - RUNTIME_PARAMETER
+    - RUNTIME_PARAMETER
+    - RUNTIME_PARAMETER
```

Comment:
soooo according to Liz and other examples, the actual number here doesn't matter, its more that the values are read as sub-elements of validHosts, which is indicated by the - 

ie it could have 1 or 4 `- RUNTIME_PARAMETER ` and it shouldn't matter